devices.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "id": "Nexusplayer",
+    "adb": {
+      "ro.product.device": [ "fugu" ],
+      "ro.bootloader": ["FUGU-02.11"],
+      "ro.build.id": ["MMB29T", "LMY48M", "LMY47D"]
```

Comment:
Yes, but it's not enough. Are you sure a build done with your tree will work with those L and M releases ?

I'm not sure that blobs will be compatible between M and L for example